spec/mainadditions.html

```diff
@@ -212,91 +208,167 @@ <h1>
+          <ins>Time zone identifiers are compared using ASCII-case-insensitive comparisons.</ins>
+        </p>
+        <p>
+          A <dfn variants="primary time zone identifiers">primary time zone identifier</dfn> is the preferred identifier for an available named time zone.
+          A <dfn variants="non-primary time zone identifiers">non-primary time zone identifier</dfn> is an identifier for an available named time zone that is not a primary time zone identifier.
+          An <dfn variants="available named time zone identifiers">available named time zone identifier</dfn> is either a primary time zone identifier or a non-primary time zone identifier.
+          Each available named time zone identifier is associated with exactly one available named time zone.
+          Each available named time zone is associated with exactly one primary time zone identifier and zero or more non-primary time zone identifiers.
+        </p>
+
+        <ins class="block">
+        <p>
+          <emu-xref href="#sec-temporal-timezone-constructor">The `Temporal.TimeZone` constructor</emu-xref>, when called with an available named time zone identifier as the argument, will return a valid `Temporal.TimeZone` object.
+          When called with any other string, it will throw a *RangeError* exception.
+        </p>
+        </ins>
+        <p>
+          ECMAScript implementations must support an available named time zone with the identifier *"UTC"*, which must be the primary time zone identifier for the UTC time zone.
+          In addition, implementations may support any number of other available named time zones.
+        </p>
+        <p>
+          Implementations that follow the requirements for time zones as described in the ECMA-402 Internationalization API specification are called <dfn>time zone aware</dfn>.
+          Time zone aware implementations must support available named time zones corresponding to the Zone and Link names of the IANA Time Zone Database, and only such names.
+          In time zone aware implementations, a primary time zone identifier is a Zone name, and a non-primary time zone identifier is a Link name, respectively, in the IANA Time Zone Database except as specifically overridden by AvailableNamedTimeZoneIdentifiers as specified in the ECMA-402 specification.
+          Implementations that do not support the entire IANA Time Zone Database are still recommended to use IANA Time Zone Database names as identifiers to represent time zones.
+        </p>
+      </emu-clause>
+
+      <emu-clause id="sec-availablenamedtimezoneidentifiers" type="implementation-defined abstract operation">
```

Comment:
This AO is identical to the AO in 262, except the minor editorial change of using SortStringListByCodeUnit. Nothing to see here. Note that the 402 part of the spec overrides this AO and makes significant changes to the 262 version, because the 402 version uses the IANA TZDB while the 262 version is generic.

Reply:
Could you explain what you mean?  This text here (in mainadditions.html) is the 262 version, and the one in intl.html overrides it. What's the subtractive part?

Reply:
The subtractive part is non-IANA identifiers being allowed by 262 but not by 402. But I just confirmed that to be the current state of Temporal, so changing it wouldn't belong in this PR anyway.